extensions/amp-viz-vega/0.1/amp-viz-vega.js

```diff
@@ -193,7 +193,7 @@ export class AmpVizVega extends AMP.BaseElement {
     const parsePromise = new Promise((resolve, reject) => {
       this.vega_.parse.spec(this.data_, (error, chartFactory) => {
```

Comment:
likely but can't guarantee it, since it comes from a third-party and without audit of their code, who knows what cases they throw what kind of errors. Can we check and wrap?
`error = typeof error == 'Error' ? error : new Error(error);`